styles.css

```diff
@@ -0,0 +1,414 @@
+    display: block;
+}
+
+input{
+    width: 100%;
+    height:2.3rem;
+    padding: 0 14px;
+    border-radius: 2px 0 0 2px;
+    border-style: none;
+    
+}
+
+img.deal-img{
+    height: 170px;
+    width: 170px;
+}
+
+.login-link{
+    background-color: #fff;
+    color:#2874f0;
+    height:100%;
+    padding: 8px 40px;
+    border-radius: 2px;
+}
+
+.main-content{
+    margin: 0;
+}
+
+.more-dropdown, .cart-text{
+    margin: 0 7%;
+    cursor: pointer;
+}
+
+.more-dropdown{
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-items:hover .dropdown-icon  i{
+    rotate: 180deg;
+    transition: 0.3s;
+}
+
+.more-dropdown:hover .more-dropdown-content{
+    display: block;
+}
+
+.more-dropdown-content{
+    display: none;
+    position: absolute;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index:1;
+    background-color: #f9f9f9;
+    font-size: small;
+    width:200px;
+}
+
+.more-dropdown-content a{
+    color:black;
+    text-decoration: none;
+    display: block;
+    padding: 12px 16px;
+    text-align: left;
+}
+
+.more-dropdown a:hover  {
+    background-color: #f1f1f1;
+}
+
+.more-dropdown-icons{
+    color: #2874f0;
+    margin-right:10px;
+}
+
+.navbar{
+    background-color: #2874f0;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    padding: 0 20%;
+}
+
+.nav-left{
+    display: flex;
+    width:60%;
+} 
+
+.nav-right{
+    display: flex;
+    justify-content: space-around;
+}
+
+.navbar-item{
+    padding: 0.5em;
+    display: block;
+    color:#fff;
+    margin: 0.5% 2%;
+}
+
+.other-deals-section{
+    margin: 1% 5%;
+    box-shadow:0 3px 10px rgba(0,0,0,0.2);
+}
+
+.other-deals-header-text{
+    font-weight: bold;
+    font-size: 20px;
+    margin:4px;
+}
+
+.plus{
+    color:#ffe500;
+}
+
+.search-bar{
+    width:100%;
+}
+
+.search-btn{
+    color:#2874f0;
+    background-color: #fff;
+    border-style: none;
+    cursor: pointer;
+}
+
+.search-input{
+    display: flex;
+}
+
+.shop-now{
+    margin: 5px 0 10px 2px;
+    color:#726b6b;
+}
+
+.timer-icon{
+    color: #2874f0;
+    padding: 0 5px;
+    margin-left:3px;
+}
+
+.time-remaining{
+    color:#726b6b;
+
+}
+
+.view-all-btn{
+    padding: 12px 22px;
+    background-color: #2874f0;
+    color: white;
+    outline: none;
+    font-weight: 500;
+    border: none;
+    cursor: pointer;
+}
+
+.view-all-btn:hover{
+    background-color:#235fc0 ;
+}
+
+/* Media Queries */
+@media (max-width:1640px) {
+    .deals-and-ad-section{
+        justify-self: center;
+        justify-content: space-between;
+    }
+
+    .deals{
+        width:80%;
+    }
+    
+}
+
+@media (max-width:1500px){
+    img.adv-img{
+        width: 220px;
+    }
+}
+
+@media (max-width:1500px){
+    img.adv-img{
+        width: 200px;
+    }
+}
+
+@media (max-width:1160px){
+    img.adv-img{
+        width: 180px;
+    }
+}
+
+@media (max-width:1500px){
+    img.adv-img{
+        width: 160px;
+    }
+}
+
+@media (max-width:1500px){
```

Comment:
Why are you changing same css property for same screen size for different times?

Reply:
Maybe I forgot to change the screen size at that moment, but will pay more attention to avoid such mistakes.